### PR TITLE
Add pulsar detector dashboard to repo

### PR DESCRIPTION
**Pull request by wolfstudy**
Signed-off-by: xiaolong.ran <rxl@apache.org>

image

### CODE REVIEW

**Comment by wolfstudy** [comment]
image

**Comment by jiazhai** [comment]
#30

File: dashboards.kubernetes/pulsar_detector.json [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "id": 1,
+  "links": [],
+  "panels": [
+    {
+      "cacheTimeout": null,
+      "datasource": null,
```
Comment (sijie) [inline dashboards.kubernetes/pulsar_detector.json:23]: Can you add "{{ PULSAR_CLUSTER }}" to the data source? So the data source can be replaced with the right data source when running on Kubernetes.

Comment (sijie) [reply dashboards.kubernetes/pulsar_detector.json:23]: Example: https://github.com/streamnative/apache-pulsar-grafana-dashboard/blob/master/dashboards.kubernetes/bookkeeper.json#L83

**Pull request merged**